tests/testthat/test-assertions.R

```diff
@@ -1,7 +1,8 @@
 library(admiral.test)
 
-test_that("Test 1 : `assert_has_variables` an error is thrown if a required
-           variable is missing", {
+# assert_has_variables ----
+## Test 1: error if a required variable is missing ----
+test_that("assert_has_variables Test 1: error if a required variable is missing", {
   data(admiral_dm)
```

Comment:
remove library call and replace with `admiral.test::admiral_dm`

Reply:
I have created a dataset using `tribble()`. I think this is clearer and faster.